scraper/newsscrapper/pipelines.py

```diff
@@ -1,13 +1,42 @@
-# Define your item pipelines here
-#
-# Don't forget to add your pipeline to the ITEM_PIPELINES setting
-# See: https://docs.scrapy.org/en/latest/topics/item-pipeline.html
-
-
-# useful for handling different item types with a single interface
+import json
+import requests
 from itemadapter import ItemAdapter
+from dotenv import load_dotenv
+import os
 
+load_dotenv()
+PORT_NUMBER = os.getenv("PORT_NUMBER")
 
 class NewsscrapperPipeline:
+    def __init__(self):
+        self.url = f'http://localhost:{PORT_NUMBER}/article.create'
+        self.json = []
+    
     def process_item(self, item, spider):
+        self.json.append(item)
+        print(item)
```

Comment:
can we print less stuff to stdout? This print can be removed now right?

Reply:
@AardhynLavender, It was not the problem. I found a different solution by specifying one thing in settings file. It's fixed now, and it will be applied with new changes after I merge my branch with main